src/infrastructure/gateway/limitedPartnership/LimitedPartnershipGateway.ts

```diff
@@ -3,7 +3,7 @@ import { Transaction } from "@companieshouse/api-sdk-node/dist/services/transact
 import { ApiErrorResponse } from "@companieshouse/api-sdk-node/dist/services/resource";
 import {
   LimitedPartnership,
-  LimitedPartnershipCreated
+  LimitedPartnershipResourceCreated
```

Comment:
Fix required by changes made for https://github.com/companieshouse/api-sdk-node/releases/tag/2.0.224